### PR TITLE
Update `fetch-charset-detection` to 1.0.1

### PR DESCRIPTION
**Pull request by Ndpnt**
Currently hosted on the @opentermsarchive account and awaiting publication by the original author.

See https://github.com/node-fetch/fetch-charset-detection/issues/249

### CODE REVIEW

File: src/archivist/fetcher/htmlOnlyFetcher.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
+import convertBody from '@opentermsarchive/fetch-charset-detection'; // eslint-disable-line import/no-unresolved
```
Comment (martinratinaud) [inline src/archivist/fetcher/htmlOnlyFetcher.js:1]: Do we still need the no-unresolved?

**Review by martinratinaud — APPROVED** [review]

**Comment by martinratinaud** [comment]
@Ndpnt in fact, why don't you link directly to github instead of forking? which needs more maintenance


"fetch-charset-detection": "git+https://git@github.com/node-fetch/fetch-charset-detection"

File: src/archivist/fetcher/htmlOnlyFetcher.js (hunk 1 above)
Comment (Ndpnt) [reply src/archivist/fetcher/htmlOnlyFetcher.js:1]: I guess, there are no changes on that side.

**Comment by Ndpnt** [comment]
> @Ndpnt in fact, why don't you link directly to github instead of forking? which needs more maintenance
> 
> "fetch-charset-detection": "git+https://git@github.com/node-fetch/fetch-charset-detection"

Because the github repository does not contain the built version, so it does not work (I tried).

**Review by martinratinaud — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "@accordproject/markdown-cicero": "^0.15.2",
     "@accordproject/markdown-pdf": "^0.15.2",
+    "@opentermsarchive/fetch-charset-detection": "^1.0.1",
```
[the remark this answers is not in the document]
Comment (Ndpnt) [reply package.json:45]: See reply above

**Pull request merged**